FND/Places/Addresses.rdf

```diff
@@ -133,5 +582,72 @@
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+		<skos:definition>the first line of the street address</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.gleif.org/en/about-lei/common-data-file-format/lei-cdf-format/lei-cdf-format-version-2-1</fibo-fnd-utl-av:adaptedFrom>
+	</owl:DatatypeProperty>
+	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-plc-adr;hasAddressLine2">
+		<rdfs:label>has address line 2</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+		<skos:definition>the second line of the street address</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.gleif.org/en/about-lei/common-data-file-format/lei-cdf-format/lei-cdf-format-version-2-1</fibo-fnd-utl-av:adaptedFrom>
+	</owl:DatatypeProperty>
+	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-plc-adr;hasAddressLine3">
+		<rdfs:label>has address line 3</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+		<skos:definition>the third line of the street address</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.gleif.org/en/about-lei/common-data-file-format/lei-cdf-format/lei-cdf-format-version-2-1</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote>This element SHALL be omitted if address line 2 is omitted.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:DatatypeProperty>
+	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-plc-adr;hasAttentionLine">
+		<rdfs:label>has attention line</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+		<skos:definition>an optional, free text address line denoting the intended recipient</skos:definition>
+	</owl:DatatypeProperty>
+	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-plc-adr;hasMailRouting">
+		<rdfs:label>has mail routing</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+		<skos:definition>an optional, free text address line containing explicit routing information (this elements&apos;s presence indicates that this address is a routing / &apos;care of&apos; address)</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.gleif.org/en/about-lei/common-data-file-format/lei-cdf-format/lei-cdf-format-version-2-1</fibo-fnd-utl-av:adaptedFrom>
+	</owl:DatatypeProperty>
+	
+	<owl:DatatypeProperty rdf:about="&fibo-fnd-plc-adr;hasPostalCode">
+		<rdfs:label>has postal code</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+		<skos:definition>the postal code of this address as specified by the local postal service</skos:definition>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.gleif.org/en/about-lei/common-data-file-format/lei-cdf-format/lei-cdf-format-version-2-1</fibo-fnd-utl-av:adaptedFrom>
+	</owl:DatatypeProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-plc-adr;hasPostcode">
```

Comment:
So - your preference is to call the text version "hasTextPostalCode"?  or to call the reified one, hasIndividualPostalCode?  I've done the latter, because we don't use the class in our examples yet, but if you have a better suggestion, let me know.